src/lazerscripts/stage1/stage1boss.coffee

```diff
@@ -534,14 +534,15 @@ class Stage1BossRocket extends EntityScript
             lightness: 1.0
             gravity: (Math.random() * .2)
             vertical: 0
-          ->
+          (p, fd) ->
+            mul = fd.dt / (1000 / Crafty.timer.FPS())
```

Comment:
Make this a bit cleaner, maybe like `fd.dt / 1000` and updating the things multiplied to a value per second